packages/client-core/src/systems/WidgetUISystem.ts

```diff
@@ -158,7 +159,7 @@ const execute = () => {
   }
   for (const action of registerWidgetQueue()) {
     const widget = RegisteredWidgets.get(action.id)!
-    setLocalTransformComponent(widget.ui.entity, widgetMenuUI.entity)
+    setComponent(widget.ui.entity, EntityTreeComponent, { parentEntity: widgetMenuUI.entity })
```

Comment:
widgets are not working after changing this

Reply:
added a setComponent(widget.ui.entity, LocalTransformComponent) to here which should fix that